tests/__tests__/refmterr/refmterr_test.re

```diff
@@ -160,59 +159,33 @@ let forEachTest =
       | "Win32" => "| sed -E \"s/([A-Za-z0-9])\\\\\\\\([A-Za-z0-9])/\\\\1\\\\/\\\\2/g\" | sed -E \"s/\\\\r//g\""
       | _ => ""
       };
+
+    Pastel.(setMode(Terminal));
+
     ignore(
       Sys.command(
         Printf.sprintf(
-          "(%s) 2>&1 | berror.exe --path-to-refmttype refmttype %s > %s",
+          "(FORCE_COLOR=true %s) 2>&1 | berror.exe --path-to-refmttype refmttype %s > %s",
```

Comment:
Looks like the snapshots still don't have the ANSI sequences, I think you want

```"(%s) 2>&1 | FORCE_COLOR=true berror.exe --path-to-refmttype refmttype %s > %s"```

if we still want the ANSI escape sequences (the first bit of the command is actually just compiling the source file we want to run refmt on I believe, we want to make sure that refmterr itself (berror.exe) picks up the environment variable . There isn't currently a flag for forcing human readable, which is unfortunate.

@jordwalke might have an opinion on whether we want to keep the ANSI in our snapshots or not

Reply:
I'm pretty sure we want to keep the ANSI on the snapshots. I would revert the  snapshots and go for: 

```reason
Pastel.(
  useMode(Terminal),
  () =>
    ignore(
      Sys.command(
        Printf.sprintf(
          "(%s) 2>&1 | FORCE_COLOR=true berror.exe --path-to-refmttype refmttype %s > %s",
          cmd,
          windowsCompatibilityPipe,
          actualOutputName,
        ),
      ),
    ),
);
```

Forcing the color and using the mode terminal.

Reply:
Sounds good on keeping the ANSI, I don't think the useMode actually does anything because the berror.exe is using a "different copy" of Pastel at runtime, but the FORCE_COLOR works and is sufficient (under the hood it forces Terminal mode) so all we should need is

```reason
ignore(
      Sys.command(
        Printf.sprintf(
          "(%s) 2>&1 | FORCE_COLOR=true berror.exe --path-to-refmttype refmttype %s > %s",
          cmd,
          windowsCompatibilityPipe,
          actualOutputName,
        ),
      ),
    )
```